_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -183,7 +183,23 @@ And on Debian/Ubuntu installations with Passenger, run:
 
     service apache2 restart
 
-#### Optional Step 5 - Cleanup of RPMs and repositories
+#### Optional Step 5 - Reclaim database space
+
+After database migrations, some space can sometimes be reclaimed. It's good idea to perform database vacuum for postgres on a regular basis as well rather than relying on autovacuum feature. For PostgreSQL do the following:
+
+    foreman-maintain service stop
```

Comment:
Do we now ship foreman-maintain by default with foreman? thinking about it, perhaps this procedure should actually go into f-m? @mbacovsky

Reply:
@ekohl could you please put some light on how we ship f-m with Foreman currently?
@tbrisker, goos idea. I filed new RFE to improve upgrades in f-m https://projects.theforeman.org/issues/25928

Reply:
We have it in the repositories but currently nothing in vanilla Foreman assumes it's being used. It's also possible to do an online vacuum so I don't see why it's needed to stop all services.

Reply:
This is VACUUM FULL (see the `-f` option) which requires exclusive lock and it's better when database is not under heavy load. I can probably point out that in the text.

Reply:
AFAIK there's also `--exclude` so you don't need to start postgresql afterwards. That said, I'd split it. First just state how to do a vacuum, then that foreman-maintain can be used to stop the services to improve the load issue.

Reply:
`foreman-maintain service stop --exclude <database>` :)